robots/flakefinder/index_test.go

```diff
@@ -132,6 +133,7 @@ var _ = Describe("index.go", func() {
 						"672h": "",
 						"168h": "",
 						"024h": "flakefinder-2019-08-23-024h.html",
+						"001h": "",
```

Comment:
Why are these changes still there?

Reply:
They are not? Did you have a look at an old commit?

Reply:
:+1 that was it.